### PR TITLE
Got rid of old files.

### LINKED ISSUE #2668: General repository cleanup
### User Story

As a developer, I want to have a clean repository with no old files and clean structure.

### Description

* remove irrelevant files (verify they are not used anymore): (e.g. `CHANGELOG.md`, ..)
* verify if `shim.js` is still used, remove if not
* remove `env/test` and simplify `env/dev/user.clj` accordingly
* migrate `docs` to wiki.status.im

*Type*: Bug

### PR DESCRIPTION
**Pull request by ghost**
fixes #2668

### Summary:
Removed old and unused files.

### Steps to test:
- Usual testing, compiling and running does not fail

status: ready

### CODE REVIEW

**Comment by jeluard** [comment]
@poiga Have you migrated the doc to the wiki?

**Comment by poiga** [comment]
@jeluard The docs are in the Wiki

**Comment by jeluard** [comment]
@poiga Can you share a link?

**Comment by poiga** [comment]
@jeluard 
[https://wiki.status.im/Frequently_Asked_Questions](https://wiki.status.im/Frequently_Asked_Questions)
[https://wiki.status.im/FAQ_status-react](https://wiki.status.im/FAQ_status-react)

I think I am merging those two pages as they are very same sort of thing...

**Comment by ghost** [comment]
@jeluard [https://wiki.status.im/Frequently_Asked_Questions](https://wiki.status.im/Frequently_Asked_Questions)

File: env/dev/env/config.cljs [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 (def figwheel-urls {
                     :ios "ws://localhost:3449/figwheel-ws"
-                    :android "ws://10.0.2.2:3449/figwheel-ws"
+                    :android "ws://localhost:3449/figwheel-ws"
```
Comment (jeluard) [inline env/dev/env/config.cljs]: Can you exclude this change?

**Review by jeluard — APPROVED** [review]

**Comment by poiga** [comment]
@jeluard All done, but that Jenkins build stopped building just because that one file... Strange.

**Comment by poiga** [comment]
@jeluard all done. Please check!

**Comment by oskarth** [comment]
Adjusting reviewers so it actually gets reviewed (https://en.wikipedia.org/wiki/Diffusion_of_responsibility)

**Review by oskarth — COMMENTED** [review]
LGTM but obviously need to test build steps etc.

**Review by oskarth — APPROVED** [review]

**Comment by oskarth** [comment]
^ @jeluard @yenda

**Review by jeluard — APPROVED** [review]

**Comment by poiga** [comment]
@yenda could you please review this ;)

**Comment by oskarth** [comment]
@yenda @jeluard Feel free to merge straight away if you are happy with this and CI passes.

**Review by yenda — APPROVED** [review]

**Comment by oskarth** [comment]
@yenda You can go ahead and merge too if you are happy with it and CI passes :)

**Comment by yenda** [comment]
@poiga Hi thanks for your contribution! Could you please rebase your branch on our develop so that we can run the CI on it before merging ?

**Comment by poiga** [comment]
@yenda @oskarth @jeluard Just rebased.

**Pull request merged**